internal/fs/wrappers/monitoring.go

```diff
@@ -42,7 +53,20 @@ func init() {
 			Aggregation: view.Sum(),
 			TagKeys:     []tag.Key{tags.FSOp, tags.FSError},
 		},
-	); err != nil {
+		&view.View{
+			Name:        "fs/request_error_count",
```

Comment:
keep the names consistent with the existing naming.
Rename it to ops_error_count and ops_latency